src/derivatives/05-diffusion-derivatives.md

```diff
@@ -41,83 +41,81 @@ which has its own unique requirements with respect to:
-            <source_keywords>[_space-<space>]_model-<label>_param-<param1>_model.json
-            <source_keywords>[_space-<space>]_model-<label>_param-<param2>_model.nii[.gz]
-            <source_keywords>[_space-<space>]_model-<label>_param-<param2>_model.json
-            <source_keywords>[_space-<space>]_model-<label>_model.json
-
-            [<source_keywords>[_space-<space>]_model-<label>_param-<parama>_mdp.nii[.gz]]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<parama>_mdp.json]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<paramb>_mdp.nii[.gz]]
-            [<source_keywords>[_space-<space>]_model-<label>_param-<paramb>_mdp.json]
-```
+            sub-01_model-DTI_param_Sxx_dwimap.nii.gz
+            sub-01_model-DTI_param_Sxy_dwimap.nii.gz
+            sub-01_model-DTI_param_Sxz_dwimap.nii.gz
+            sub-01_model-DTI_param_Syy_dwimap.nii.gz
+            sub-01_model-DTI_param_Syz_dwimap.nii.gz
+            sub-01_model-DTI_param_Szz_dwimap.nii.gz
+            sub-01_model-DTI_dwimap.json
+            sub-01_model-DTI_param-FA_dwimap.nii.gz
+            sub-01_model-DTI_param-MD_dwimap.nii.gz
 
--   Files "`<source_keywords>[_space-<space>]_model-<label>_param-<param*>_model.nii[.gz]`"
-    provide data corresponding to the various requisite [model fit parameters](#paramdef-fit).
+```
 
-    In cases where *all* [model fit parameters](#paramdef-fit) are contained within a single image
-    file, entity "`_param-`" MUST NOT be included; e.g.:
-        ```Text
-        <pipeline_name>/
-            sub-<participant_label>/
-                dwi/
-                    <source_keywords>[_space-<space>]_model-<label>_model.nii[.gz]
-                    <source_keywords>[_space-<space>]_model-<label>_model.json
-        ```
+In cases where a set of different parameters are contained within a single image
+file, entity "`_param-`" MUST NOT be included; e.g.:
+    ```Text
+    <pipeline_name>/
+        sub-<participant_label>/
+            dwi/
+                <source_keywords>[_space-<space>]_model-<label>_dwimap.nii[.gz]
+                <source_keywords>[_space-<space>]_model-<label>_dwimap.json
+    ```
 
--   File "`<source_keywords>[_space-<space>]_model-<label>_model.json`"
-    provides basic model information and [input parameters](#paramdef-input).
+Example:
 
--   OPTIONAL images "`<source_keywords>[_space-<space>]_model-<label>_param-<parama>_mdp.nii[.gz]`"
-    may be defined, in order to provide additional [model-derived parameters](#paramdef-derived).
+```Text
+my_tensors/
+    sub-01/
+        dwi/
+            sub-01_model-DTI_dwimap.nii.gz # Contains the six diffusion tensor components
+            sub-01_model-DTI_dwimap.json
+            sub-01_model-DTI_param-FA_dwimap.nii.gz
+            sub-01_model-DTI_param-MD_dwimap.nii.gz
+            sub-01_model-DKI_dwimap.nii.gz # Contains the fifteen kurtosis tensor components
```

Comment:
Personally I don't think it makes sense to have valid interpretation of a DKI model being predicated on the contents of a DTI model that is explicitly encoded as being a different model. The fact that these data are split across different data files with different `_model-` entity values to me means they are independent.

I never committed to how to deal with kurtosis because even for the tensor it didn't reach the state I wanted (#74). With DKI I would want the logic of #74 to be extended to cover the rank-4 tensor also being a data representation, with the specification indicating the ordering of tensor coefficients across image volumes.

There might be a debate to be had over whether the six rank-2 coefficients should be in one file and the fifteen extra coefficients in a different file, versus all coefficients encoding the rank-4 tensor being in a single image (or having the specification facilitate both alternatives). But the six rank-2 coefficients should not IMO be distanced so far from the kurtosis model fit so as to be attributed to an entirely different model. For one it wouldn't be possible to calculate from just the 15 kurtosis coefficients values of FA and MD as shown here.

Reply:
Sorry - this was just an error on my part. I got carried away wanting to show how this would generalize to DKI as well. This should have been "the *twenty one* tensor components" - I was not envisioning separating the six from the fifteen in some over clever way. Here, I was imagining a situation where someone runs a DTI fit and then also a DKI fit and has both the results from the DTI and the DKI side-by-side. I'll try to clarify.